### PR TITLE
Lift out globals and paths to central location

### PR DESCRIPTION
**Pull request by collijk**
- Collected all globals into a single top-level globals module
- Replaced raw strings with globals wherever possible
- Collected all paths and path-finding into a single globals module
- Cleanup of some duplicated logic

I'm trying to do three things:
1. Make sure I'm familiar with the current data processing pipeline and where stuff lives.
2. Gather all the things I'm going to need for the results processing pipeline in a place that makes sure it stays consistent with the input processing pipeline.
3. Start pushing down some of the technical debt from code duplication.  The project is going to slow down and we might not look at this code for a bit and then need to come back and modify several pieces after feedback from clients.  The fiddly inconsistencies across files will eat our lunch if we step away for a few weeks.

For the pieces you wrote, make sure I didn't introduce any errors.  Almost everything here is just naming changes, and I'll point out where it's not below.

### CODE REVIEW

File: src/vivarium_csu_ltbi/components/disease.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from vivarium_public_health.disease import DiseaseState, DiseaseModel, SusceptibleState, RateTransition
-from .names import *
+
```
Comment (collijk) [inline src/vivarium_csu_ltbi/components/disease.py:2]: Namespace change only in this file

File: src/vivarium_csu_ltbi/components/names.py [hunk 2]
```diff
@@ -1,28 +0,0 @@
-# disease name
```
Comment (collijk) [inline src/vivarium_csu_ltbi/components/names.py:1]: Merged into vivarium_csu_ltbi.globals

File: src/vivarium_csu_ltbi/components/observers.py [hunk 3]
```diff
@@ -124,44 +126,29 @@ def __init__(self):
...
-                                    self.causes)
-        unexposed_deaths = {f'{k}_unexposed_to_hhtb': v for k, v in unexposed_deaths.items()}
-        unexposed_ylls = get_years_of_life_lost(unexposed, self.config.to_dict(), self.start_time, self.clock(),
-                                              self.age_bins, self.life_expectancy, self.causes)
-        unexposed_ylls = {f'{k}_unexposed_to_hhtb': v for k, v in unexposed_ylls.items()}
-
-        metrics.update(unexposed_person_time)
-        metrics.update(unexposed_deaths)
-        metrics.update(unexposed_ylls)
+        measure_getters = (
```
Comment (collijk) [inline src/vivarium_csu_ltbi/components/observers.py:137]: This is the only significant refactor in this PR.  Sorry, should have saved it for a separate PR.  No logic changes here, just converting to a loop.

File: src/vivarium_csu_ltbi/components/observers.py [hunk 4]
```diff
@@ -179,28 +166,25 @@ def __init__(self):
...
-                                                                   self.causes)
-        unexposed_ylds_this_step = {f'{k}_unexposed_to_hhtb': v for k, v in unexposed_ylds_this_step.items()}
-        self.years_lived_with_disability.update(unexposed_ylds_this_step)
+        self.update_metrics(pop)
 
         pop.loc[:, 'years_lived_with_disability'] += self.disability_weight(pop.index)
         self.population_view.update(pop)
+
+    def update_metrics(self, pop):
+        exposure_category = self.household_tb_exposure(pop.index)
```
Comment (collijk) [inline src/vivarium_csu_ltbi/components/observers.py:180]: same as other refactor in this file

File: src/vivarium_csu_ltbi/data/cli.py [hunk 5]
```diff
@@ -3,10 +3,10 @@
 
 import click
 from loguru import logger
-
 from vivarium import Artifact
 from vivarium_cluster_tools.psimulate.utilities import get_drmaa
-from vivarium_csu_ltbi.data.globals import COUNTRIES, formatted_country
+
+from vivarium_csu_ltbi import globals as ltbi_globals
```
Comment (collijk) [inline src/vivarium_csu_ltbi/data/cli.py:10]: I also changed country to location pretty much everywhere. Some places we were using country, some location, and some country name.  In practice, we might get requests for nationals and subnationals in the same project and location is the term of art around the institute for all such things.

File: src/vivarium_csu_ltbi/data/globals.py [hunk 6]
```diff
@@ -1,15 +0,0 @@
-
```
Comment (collijk) [inline src/vivarium_csu_ltbi/data/globals.py:1]: Merged into vivarium_csu_ltbi.globals

File: src/vivarium_csu_ltbi/data/household_tb_paths.py [hunk 7]
```diff
@@ -1,38 +0,0 @@
-from pathlib import Path
```
Comment (collijk) [inline src/vivarium_csu_ltbi/data/household_tb_paths.py:1]: Merged into vivarium_csu_ltbi.paths

File: src/vivarium_csu_ltbi/data/ltbi_incidence_model.py [hunk 8]
```diff
@@ -1,28 +1,27 @@
 import pandas as pd
 import numpy as np
-import dismod_mr
```
Comment (collijk) [inline src/vivarium_csu_ltbi/data/ltbi_incidence_model.py:3]: Hid these imports in the functions so users don't get unneccesary import errors.

File: src/vivarium_csu_ltbi/external_data/ltbi_hh_tb_exposure_calculation.py [hunk 9]
```diff
@@ -1,111 +0,0 @@
-import pandas as pd
```
Comment (collijk) [inline src/vivarium_csu_ltbi/external_data/ltbi_hh_tb_exposure_calculation.py:1]: this was yongquan's original script and a duplicate of cody's refactor

**Review by halfhorst — APPROVED** [review]

**Pull request merged**